includes/settings/class-wcpdf-settings-debug.php

```diff
@@ -66,6 +67,8 @@ public function output( $active_section ) {
 				$this->display_numbers();
 				break;
 		}
+
+		do_action( 'wpo_wcpdf_settings_debug_after_output', $active_section );
```

Comment:
Maybe you could include this in the `switch` statement?

```php
default:
	do_action( 'wpo_wcpdf_settings_debug_section_output', $active_section );
	break;
```

Reply:
I thought for more flexibility and usability, it could be better to put it outside of the switch statement. Is it better as a switch case?

Reply:
You can have both.